.circleci/config.yml

```
@@ -25,7 +25,7 @@ jobs:
 
   test_js:
     docker:
-      - image: "circleci/node:16-browsers"
+      - image: "circleci/python:3.9-node-browsers"
```

Note to self: this works for the sake of running the JS tests, but it relies on the venv and dependencies Glean creates automatically. Is there a better solution?

---

Drive-by comment: Circle is moving its image repo to use the `cimg` prefix, so might be good to move to that now (assuming it's the same behaviour as the `circleci` versions
```suggestion
      - image: "cimg/python:3.9-node-browsers"
```

---

@stevejalim i tried this but the `node-browsers` suffix isn’t yet supported in their new infra